ansible/files/walg_helper_scripts/wal_fetch.sh

```diff
@@ -0,0 +1,12 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+# Fetch the WAL file and temporarily store them in /tmp
+sudo -u wal-g wal-g wal-fetch $1 /tmp/$1 --config /etc/wal-g/config.json 
```

Comment:
quoting etc; just verify with shellcheck

Reply:
Can we nest the file under a more specific dir under /tmp ?

e.g. /tmp/supabase-restores or similar? And then verify the same before ownership reassignment